testing/citest/tests/openstack_smoke_test.py

```diff
@@ -0,0 +1,258 @@
+# Copyright 2017 Veritas Technologies, LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Integration test to see if the security group creation process is working for
+the Spinnaker OpenStack integration.
+
+See testable_service/integration_test.py and spinnaker_testing/spinnaker.py
+for more details.
+
+The test will use the spinnaker configuration parameters from the server
+endpoint (gate) to determine the managed project it should verify, and to
+determine the spinnaker account name to use when sending it commands.
+
+Sample Usage:
+    PYTHONPATH=$CITEST_ROOT:$CITEST_ROOT/spinnaker \
```

Comment:
In practice I use virtualenv and have citest installed there so it's in the path already. The testing/citest/README recommends that. So I think thing CITEST_ROOT needs to be in the path.
CITST_ROOT/spinnaker is old from when this was in the citest repository. These are now in spinnaker/testing/citest. LIkewise below path is wrong.

PYTHONPATH=testing/citest python testing/citest/tests/openstack_smoke_test.py --native_hostname=...